cmd/nodeagent/nodeagentmanager.go

```diff
@@ -54,6 +56,7 @@ func NewNodeAgentManager(dynamicClient dynamic.Interface, nodeName string, logEv
 		logEveryHeartbeat: logEveryHeartbeat,
 	}
 	n.startWatch()
+	n.startHeartbeatTimeCheckerLoop()
```

Comment:
I assume this was named after the `startWatch` above, but I feel that they both need a better name, maybe `runWatch` and `runHeartbeatTimeCheckerLoop`?